src/server/scripts/Spells/spell_priest.cpp

```diff
@@ -3079,6 +3080,45 @@ class spell_pri_shadow_mend_periodic_damage : public AuraScript
     }
 };
 
+// 32379 - Shadow Word: Death
+class spell_pri_shadow_word_death : public SpellScript
+{
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellEffect
+        ({
+            { SPELL_PRIEST_SHADOW_WORD_DEATH, EFFECT_1 },
+            { SPELL_PRIEST_SHADOW_WORD_DEATH, EFFECT_2 },
+            { SPELL_PRIEST_SHADOW_WORD_DEATH, EFFECT_4 }
+        });
+    }
+
+    void HandleEffectHit(SpellEffIndex /*effIndex*/)
+    {
+        Unit* caster = GetCaster();
+        Unit* target = GetHitUnit();
+
+        uint64 hitDamage = GetHitDamage();
+
+        if (target->GetHealthPct() < GetEffectInfo(EFFECT_1).BasePoints)
```

Comment:
never BasePoints, CalcValue(caster) in case there's anything that might be modifying this effect

Reply:
there might be an aura of whatever sort that increases the threshold, which has been fairly common in the past